assets/javascripts/lib/get-debounce.js.es6

```diff
@@ -0,0 +1,11 @@
+import discourseDebounce from "discourse-common/lib/debounce";
+import { debounce } from "@ember/runloop";
+
+// TODO: Remove this file and use discouseDebounce after the 2.7 release.
+export default function () {
+  if (discourseDebounce === undefined) {
+    return debounce;
+  }
+
+  return discourseDebounce;
+}
```

Comment:
How about renaming this file to `debounce.js.es6` and exporting a function instead of a function factory?

```suggestion
const debounceFunction = discourseDebounce || debounce;
export default debounceFunction;
```

All call sites could then remain unchanged.